### PR TITLE
Bug 1892382: Fix operator exited message

### PR DESCRIPTION
**Pull request by awgreene**

### CODE REVIEW

**Review by ankitathomas — APPROVED** [review]
/lgtm

File: pkg/status/status.go [hunk 1]
```diff
@@ -188,17 +181,7 @@ func (r *reporter) monitorClusterStatus() {
...
-			msg := "The operator has exited"
-			conditionListBuilder := clusterStatusListBuilder()
-			conditionListBuilder(configv1.OperatorProgressing, configv1.ConditionFalse, msg, reason)
-			conditionListBuilder(configv1.OperatorAvailable, configv1.ConditionFalse, msg, reason)
-			statusConditions := conditionListBuilder(configv1.OperatorDegraded, configv1.ConditionFalse, msg, reason)
-			statusErr := r.setStatus(statusConditions)
-			if statusErr != nil {
-				log.Error("[status] " + statusErr.Error())
-			}
+			log.Info("[status] Operator no longer reporting status")
```
Comment (anik120) [inline pkg/status/status.go:184]: Do we not want to alert cluster operator if the operator exists? That doesn't seem right

Comment (awgreene) [reply pkg/status/status.go:184]: Trevor King requested that we don't set off a telemeter alarm on graceful exits.

Comment (anik120) [reply pkg/status/status.go:184]: Gotcha :+1:

**Review by dinhxuanvu — APPROVED** [review]
/lgtm

**Comment by tbuskey** [comment]
[https://bugzilla.redhat.com/show_bug.cgi?id=1892382 ](https://bugzilla.redhat.com/show_bug.cgi?id=1892382 )is in POST, not ON_QA

**Pull request merged**